Subaharish/HTML BASICS/assignment3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Registration</title>
+    </head>
+    <body>
+        <h1>Register</h1>
+        <form>
+            First name:
+            <input type="text" name="Firstname"  placeholder="Firstname" required>
+            Last name:
+            <input type="text" name="Lastname" placeholder="Lastname" required>
+            <br>
+            <br>
+            Gender:
+
+            <label for="Male">Male:</label>
+            <input id="Male" type="radio" name="gender" value="male" checked>
+            <label for="Female">Female:</label>
+            <input id="Female" type="radio" name="gender" value="female">
+            <label for="Other">Other:</label>
+            <input id="Other" type="radio" name="gender" value="other" >
+            <br>
+            <br>
+            Email:
+            <input type="email" name="email" placeholder="Email" required>
+            Password:
+            <input type="password" name="password" minlength="5" maxlength="15" required>
+            Birthday:
+            <input type="date" name="birthday" required><br>
```

Comment:
Missing autocomplete. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)